Dockerfile.Nextcloud

```diff
@@ -0,0 +1,11 @@
+ARG NEXTCLOUD_VERSION=29.0.4
+
+FROM nextcloud:${NEXTCLOUD_VERSION}
+
+RUN apt update && apt install -y less vim telnet iputils-ping
+RUN curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar \
```

Comment:
thats my fault. We don't need the wp-cli in the nextcloud container. Its a copy paste mistake from the wordpress container.